parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -682,21 +689,23 @@ mod tests {
 		let beneficiary_address: [u8; 20] = hex!("2000000000000000000000000000000000000000");
 
 		let assets: MultiAssets = vec![MultiAsset {
-			id: Concrete(X1(AccountKey20 { network: Some(network), key: token_address }).into()),
+			id: Concrete(X1(AccountKey20 { network: None, key: token_address }).into()),
 			fun: Fungible(1000),
 		}]
 		.into();
 		let filter: MultiAssetFilter = Wild(All);
 
 		let message: Xcm<()> = vec![
 			UnpaidExecution { weight_limit: Unlimited, check_origin: None },
-			ReserveAssetDeposited(assets),
-			ClearOrigin,
+			WithdrawAsset(assets),
 			DepositAsset {
 				assets: filter,
-				beneficiary: X1(AccountKey20 { network: Some(network), key: beneficiary_address })
-					.into(),
+				beneficiary: X1(AccountKey20 { network: None, key: beneficiary_address }).into(),
 			},
+			SetTopic([
+				0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+				0, 0, 0, 0,
+			]),
```

Comment:
Rather use an array initializer: https://www.joshmcguigan.com/blog/array-initialization-rust/

```suggestion
			SetTopic([0u8; 32]),
```